### PR TITLE
Johnsmartco patch2

### PR DESCRIPTION
**Pull request by johnsmartco**

### CODE REVIEW

**Comment by johnsmartco** [comment]
Not quite ready for merge. I need to verify a few more steps.

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 1]
```diff
@@ -32,7 +33,20 @@ You will need storage for the backups. This topic shows the use of AWS S3 bucket
...
+helm repo update
+```
+
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "k8ssandra" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+* If you haven’t already, install the k8ssandra chart in any namespace; this command will use the `default` namespace.
```
Comment (bradfordcp) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:49]: `If you haven’t already,`

Could we rephrase this as it is the same as the previous item? I'm on the fence regarding this change request so feel free to push back.

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 2]
```diff
@@ -49,7 +63,7 @@ The first `kubectl` command above installed the cass-operator and the Prometheus
 
 ### Create secret for read/write access to an S3 bucket
 
-Before creating the k8ssandra-cluster, we need to supply credentials so that Medusa has read/write to an S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, GKE, and other bucket types. In this example, we’re using S3.
+Before creating the k8ssandra-cluster, we need to supply credentials so that Medusa has read/write to an AWS S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, GKE, and Azure buckets. Currently, K8ssandra supports S3.
```
Comment (bradfordcp) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:66]: `GKE` should be `Google Cloud Storage`.

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 3]
```diff
@@ -84,11 +98,27 @@ secret/medusa-bucket-key configured
...
+    bucketSecret: medusa-bucket-key
+    multiTenant: true
+    storage: s3
+```
+
+Example:
+
+`helm install k8ssandra-cluster-1 k8ssandra/k8ssandra-cluster -f backup-restore-values.yaml`
+
+Backup and restore operations are enabled by default. In the example YAML, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
```
Comment (bradfordcp) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:118]: > The `bucketSecret` corresponds to the secret credentials.

We should clarify that it relates to a Kubernetes Secret which contains the object store credentials. Bonus points if we illustrate what that secret should look like.

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 4]
```diff
@@ -84,11 +98,27 @@ secret/medusa-bucket-key configured
...
 
-`helm install k8ssandra-cluster-1 k8ssandra/k8ssandra-cluster --set backupRestore.medusa.bucketName=k8ssanda-bucket-dev --set backupRestore.medusa.bucketSecret=medusa-bucket-secret`
 
-The `k8ssandra-cluster` Helm chart includes cluster services and the Grafana Operator. Notice that `k8ssandra-cluster` add a number of properties in the `cassdc` datacenter.  
+The `k8ssandra-cluster` Helm chart also includes cluster services and the Grafana Operator. Notice that `k8ssandra-cluster` adds a number of properties in the `cassdc` datacenter.
```
Comment (bradfordcp) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:121]: What do we mean by "cluster services"?

**Review by bradfordcp — CHANGES_REQUESTED** [review]
Excellent PR, some minor nits and content changes. Thanks!

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 5]
```diff
@@ -13,6 +13,7 @@ This topic walks you through the steps to backup and restore Cassandra data runn
 * K8ssandra-cluster Helm chart, which we'll extend with `backupRestore` Medusa buckets for Amazon S3 integration
 * Sample files in GitHub:
   * [medusa-bucket-key.yaml](./medusa-bucket-key.yaml) to create a secret with credentials for AWS S3 buckets
+  * [backup-restore-values.yaml](./backup-restore-values.yaml) to enable Medusa (backup/restore service) and set related minimal values
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Done, thanks

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Done, thanks

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 3 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Done, thanks

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 4 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Reworded, thanks

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Done, thanks Chris.

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 6]
```diff
@@ -66,13 +80,42 @@ kubectl apply -f my-medusa-bucket-key.yaml
...
+backupRestore: 
+  medusa:
+    enabled: true
+    bucketName: k8ssandra-bucket-dev
+    bucketSecret: medusa-bucket-key
+    multiTenant: true
+    storage: s3
+```
+
+The chart's entries relate to a Kubernetes Secret, which contains the object store credentials.
```
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:98]: More specifically the `bucketSecret` property specifies the name of a secret that should contain an AWS access key. The AWS account with which the key is associated should have the permissions as described in the [Medusa docs](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md) that are required for Medusa to access the S3 bucket.

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 7]
```diff
@@ -148,7 +191,7 @@ Review the current charts that are in use, so far:
 
 ```
 NAME               	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART                  	APP VERSION
-k8ssandra         	default  	1       	2020-11-16 20:17:23.107265 -0700 MST	deployed	k8ssandra-0.23.0        3.11.7  
+k8ssandra.         	default  	1       	2021-01-16 20:17:23.107265 -0700 MST	deployed	k8ssandra-0.9.0        	3.11.7
```
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:194]: Should the period after _k8ssandra_ in the first column be there?

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 8]
```diff
@@ -66,13 +80,42 @@ kubectl apply -f my-medusa-bucket-key.yaml
...
+.
+.
+.
+```
+
+Backup and restore operations are enabled by default. In the example YAML, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
+
+The `k8ssandra` Helm chart includes the Grafana Operator. Notice that `k8ssandra` adds a number of properties in the `cassdc` datacenter.  
 
 `kubectl get cassdc dc1 -o yaml`
```
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:120]: Checking for `cassandrOperatorProgress: Ready` is not always sufficient. The CassandraDatacenter status reports several conditions. The user should also look for the following condition in the status:

```yaml
status:
  cassandraOperatorProgress: Ready
  conditions:
  ...
  - lastTransitionTime: "2021-02-03T17:04:52Z"
    message: ""
    reason: ""
    status: "True"
    type: Ready
  ...
```

**Review by jsanda — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 9]
```diff
@@ -66,13 +80,42 @@ kubectl apply -f my-medusa-bucket-key.yaml
...
+Install the `k8ssandra` chart with the following properties. You can reference the provided [backup-restore-values.yaml](./backup-restore-values.yaml) file. It contains:
+
+```
+size: 3
+backupRestore: 
+  medusa:
+    enabled: true
+    bucketName: k8ssandra-bucket-dev
+    bucketSecret: medusa-bucket-key
+    multiTenant: true
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:94]: Ok will do in the sample file and in this topic's list of its content - thanks

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 8 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:119]: Incorporated - thanks

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 6 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Incorporated - thanks John

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Done in topic and sample file

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 7 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: good catch - removed

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 10]
```diff
@@ -66,13 +80,49 @@ kubectl apply -f my-medusa-bucket-key.yaml
...
+k8ssandra-kube-prometheus-operator-85695ffb-xdq8c     1/1     Running            0          18h
+k8ssandra-medusa-operator-58586cb9c4-qctvb            1/1     Running            5          16m
+k8ssandra-reaper-operator-97754757-p7rz8              1/1     Running            11         18h
+prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running            1          18h
+prometheus-pulsar-kube-prometheus-sta-prometheus-0    2/2     Running            1          18h
+```
+
+Backup and restore operations are enabled by default. In the example YAML, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
+
+The `k8ssandra` Helm chart includes the Grafana Operator. Notice that `k8ssandra` adds a number of properties in the `cassdc` datacenter.
```
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:125]: We no longer include the Grafana operator. We will include Grafana itself though.

**Review by jsanda — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 10 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/restore-a-backup/_index.md]: Edited - thanks

**Review by jsanda — APPROVED** [review]

**Review by bradfordcp — APPROVED** [review]

File: docs/content/en/docs/topics/restore-a-backup/_index.md [hunk 11]
```diff
@@ -29,17 +30,30 @@ All other prerequisites are handled by the installed tools listed above. The sam
...
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "k8ssandra" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
 
 ### Create secret for read/write access to an S3 bucket
 
-Before creating the k8ssandra cluster, we need to supply credentials so that Medusa has read/write to an S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, GKE, and other bucket types. In this example, we’re using S3.
+Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write to an AWS S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, Google Cloud Storage, and Azure buckets. Currently, K8ssandra supports S3.
```
Comment (bradfordcp) [inline docs/content/en/docs/topics/restore-a-backup/_index.md:50]:
```suggestion
Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write to an AWS S3 bucket, which is where the backup will be stored.  Medusa currently supports local, Amazon S3, Google Cloud Storage, and Azure buckets. K8ssandra only exposes configuration for Amazon S3 at this time.
```

**Review by bradfordcp — CHANGES_REQUESTED** [review]
One small tweak, we repeated `currently` at the start of two sentences starting each other. What do you think of the suggestion?

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 11 above)
Comment (jsanda) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:50]: There are couple problems I spotted. Here is a revised version:

Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write access to an AWS S3 bucket, which is where the backup will be stored.  Medusa currently supports local, Amazon S3, Google Cloud Storage, and Azure Blob Storage.. K8ssandra exposes configuration for Amazon S3 and Google Cloud Storage at this time.

Comment (bradfordcp) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:50]: Updated suggestion with @jsanda's recommendation

```suggestion
Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write access to an AWS S3 bucket, which is where the backup will be stored. Medusa currently supports local, Amazon S3, Google Cloud Storage, Azure Blob Storage, and IBM Cloud Object Storage. K8ssandra exposes configuration for Amazon S3 and Google Cloud Storage at this time.
```

Comment (bradfordcp) [reply docs/content/en/docs/topics/restore-a-backup/_index.md:50]: @johnsmartco if this works for you commit the suggestion and we will wrap up approvals. Thanks!

**Comment by johnsmartco** [comment]
Thanks guys, I committed the suggestions (all good).  Looks like I don't have permission to merge this PR. Please go ahead and merge to main.

**Review by bradfordcp — APPROVED** [review]

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 11 above)
Comment (jsanda) [inline docs/content/en/docs/topics/restore-a-backup/_index.md]: This still says `has read/write to an AWS S3 bucket`. It should be `has read/write access to an AWS S3 bucket`.

**Review by jsanda — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/topics/restore-a-backup/_index.md (hunk 11 above)
Comment (bradfordcp) [inline docs/content/en/docs/topics/restore-a-backup/_index.md]:
```suggestion
Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write access to an AWS S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, Google Cloud Storage, and Azure buckets. Currently, K8ssandra supports S3.
```

**Review by bradfordcp — CHANGES_REQUESTED** [review]

**Comment by bradfordcp** [comment]
I've made the edits @jsanda. Can you take another look?

**Review by bradfordcp — APPROVED** [review]

**Review by jsanda — APPROVED** [review]
